### DISCUSSION TITLE
支持的效果-高斯模糊有bug,渲染不生效

### DISCUSSION THREAD

**Discussion by Xiaolianmao** [Tencent/libpag#2014] [Q&A] [answered]
PAGViewer 4.3.8.105(beta) | After Effects 2023 23.6.0(Build) |  macOS Sonoma 14.1.2

把高斯模糊的效果应用在合成图层上,并且模糊方向改成:垂直 或者改成:水平的时候在渲染, 高斯模糊的效果就会失效,pag渲染不出来
![2](https://github.com/Tencent/libpag/assets/35087840/f51087b0-2155-4592-b1b3-1ac0896ddc48)

![1](https://github.com/Tencent/libpag/assets/35087840/134db534-3fcb-4478-a6b2-0c2317ea1dda)

文件⬇️
[高斯模糊bug.zip](https://github.com/Tencent/libpag/files/13710381/bug.zip)

**Comment by jinwuwu001** [comment, marked as answer]
libpag最新版已修复该问题，待后续PAGViewer发布新版本后带上该修复
